build.sh

```diff
@@ -27,6 +27,7 @@ PRINT_USAGE() {
     echo "    --icu         : Path to ICU include folder (see example below)"
     echo "    --jobs, -j    : Multicore build (i.e. -j=3 for 3 cores)"
     echo "    --ninja, -n   : Build with ninja instead of make"
+    echo "    --test, -t    : Test build (by default Release Build)"
```

Comment:
agree with @obastemur. Alternatively, --fretest if you want something shorter? Although that has msft connotations with the usage of fre, so I'm fine with whatever here.